specification/storagesync/resource-manager/Microsoft.StorageSync/preview/2020-09-01/storagesync.json

```diff
@@ -5162,6 +5181,190 @@
         "name": "RegisteredServerAgentVersionStatus",
         "modelAsString": true
       }
+    },
+    "CloudEndpointChangeEnumerationStatus": {
+      "type": "object",
+      "description": "Cloud endpoint change enumeration status object",
+      "properties": {
+        "lastUpdatedTimestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Last updated timestamp",
+          "readOnly": true
+        },
+        "lastEnumerationStatus": {
+          "$ref": "#/definitions/CloudEndpointLastChangeEnumerationStatus",
+          "description": "Status of last completed change enumeration",
+          "readOnly": true
+        },
+        "activity": {
+          "$ref": "#/definitions/CloudEndpointChangeEnumerationActivity",
+          "description": "Change enumeration activity",
+          "readOnly": true
+        }
+      }
+    },
+    "CloudEndpointLastChangeEnumerationStatus": {
```

Comment:
just curious, why CloudEndpointLastChangeEnumerationStatus is different with CloudEndpointChangeEnumerationStatus? It seems same model just time different?

Reply:
hi @anpint, could you pls have a look?

Reply:
Hi @yungezz sorry for the delay. The "CloudEndpointLastChangeEnumerationStatus" is a "container" object that holds information about a process for our service called "change enumeration". Currently, the information it contains is about current progress of an ongoing "change enumeration" operation, and information about the previous run of this operation, that's the "LastChangeEnumerationStatus". Perhaps we could have chosen a name with less overlap, but changing the name now will also changing our cloud service implementation of the protocol, which is something we want to avoid at this point.